src/main/kotlin/nl/avisi/structurizr/site/generatr/site/views/AutoReloading.kt

```diff
@@ -0,0 +1,37 @@
+package nl.avisi.structurizr.site.generatr.site.views
+
+import kotlinx.html.*
+import nl.avisi.structurizr.site.generatr.site.asUrlRelativeTo
+import nl.avisi.structurizr.site.generatr.site.model.PageViewModel
+
+fun HEAD.script(viewModel: PageViewModel) {
+    script(
+        type = ScriptType.textJavaScript,
+        src = "../" + "/auto-reload.js".asUrlRelativeTo(viewModel.url)
+    ) { }
+}
+
+fun BODY.progress() {
+    div(classes = "is-overlay is-front-centered") {
+        progress(classes = "progress is-small mt-1 is-hidden") {
+            id = "progress"
+            value = "0"
+        }
+    }
+}
+
+fun BODY.hero() {
```

Comment:
Here, I would also prefer a name that indicates what it's for, rather than what it is. Something like:

```suggestion
fun BODY.errorMessage() {
```